src/main/java/step2/domain/winning/WinningScore.java

```diff
@@ -0,0 +1,35 @@
+package step2.domain.winning;
+
+import java.util.Arrays;
+
+public enum WinningScore {
+
+    MISS(0, 0),
+    THREE(3, 5_000),
+    FOUR(4, 50_000),
+    FIVE(5, 1_500_000),
+    SIX(6, 2_000_000_000);
+
+    private final int correctCount;
+    private final long winningAmount;
+
+    WinningScore(int correctCount, long winningAmount ) {
+        this.correctCount = correctCount;
+        this.winningAmount = winningAmount;
+    }
+
+    public static final WinningScore valueOf(int realCorrectCount) {
+        return Arrays.stream(WinningScore.values())
+                .filter(operation -> operation.correctCount == realCorrectCount)
+                .findFirst().orElse(WinningScore.MISS);
```

Comment:
```suggestion
                .filter(operation -> operation.correctCount == realCorrectCount)
                .findFirst()
                .orElse(WinningScore.MISS);
```
스트림 관련 메소드는 한 줄에 한 개만 사용해주세요!